forge-gui/res/cardsfolder/upcoming/city_of_the_daleks.txt

```diff
@@ -0,0 +1,12 @@
+Name:City of the Daleks
+ManaCost:no cost
+Types:Plane Skaro
+T:Mode$ AttackersDeclared | AttackingPlayer$ You | Execute$ TrigLoseLife | TriggerZones$ Command | TriggerDescription$ Whenever your team attacks with exactly two creatures, those creatures gain double strike until end of turn.
+SVar:TrigLoseLife:DB$ LoseLife | LifeAmount$ X | ValidTgts$ Opponent | TgtPrompt$ Select target opponent
+T:Mode$ ChaosEnsues | TriggerZones$ Command | Execute$ TrigRepeat | TriggerDescription$ Whenever chaos ensues, for each opponent, you create a 3/3 black Dalek artifact creature token with menace that attacks that opponent this turn if able. Those tokens gain haste. Sacrifice them at the beginning of the next end step.
```

Comment:
these old scheme cards are doing it wrong
the "attacks that opponent this turn if able" is not an ability of the token
it's an effect that gets applied to the game rules

you could look at the Encore implementation inside CardFactory as a better approach for educational purpose, but since this is also only another variant card I don't see this as high priority

Reply:
should be fixed but still needs testing

Reply:
Seems to work in my testing 
![image](https://github.com/Card-Forge/forge/assets/67333662/0da92008-ed7f-4450-b786-66f30b612413)